Reporter/Reporter.cs

```diff
@@ -593,16 +609,16 @@ void clear()
 		selectedLog = null;
 	}
 
-	Rect screenRect;
-	Rect toolBarRect;
+	Rect screenRect = Rect.zero;
+    Rect toolBarRect = Rect.zero;
```

Comment:
extra blank lines should be fixed here, but it's fine

Reply:
Yeah, I should have fixed those! Thank you! Just wonder if @aliessmael is going to accept this patch or not because he seems like he hasn't been on GitHub for an extended period.